src/main/java/com/github/manedev79/timesheet/TimesheetApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Configuration;
 import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
+@Configuration
```

Comment:
Are you sure we need this here? I don't see any reason why. @SpringBootApplication should be enough :)

Reply:
Not sure. Checked it and removed the annotation.